Runner/Runner.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import print_function
```

Comment:
💡 recommendation: Unt tests? they will come very handy in longer run.

Reply:
I was thinking about adding tests to this and yes I do like adding tests to this. But in this case, I feel the purpose of this script is really just to test out all the templates. I don't feel like it's worth adding